templates/distribution/manifests/aws/patches/ebs-csi-driver.yml.tpl

```diff
@@ -1,7 +1,12 @@
+{{- $module := index .spec.distribution.modules "aws" -}}
+
+---
 apiVersion: v1
 kind: ServiceAccount
 metadata:
   annotations:
-    eks.amazonaws.com/role-arn: {{ .spec.distribution.modules.aws.ebsCsiDriver.iamRoleArn }}
+    {{- if $module -}}
+    eks.amazonaws.com/role-arn: {{ $module.ebsCsiDriver.iamRoleArn }}
+    {{- end -}}
```

Comment:
why add a patch when the only change is within an "if"?

Reply:
I added this to guard against accessing a null value, should the aws key be empty.